### DISCUSSION TITLE
Graphql API

### DISCUSSION THREAD

**Discussion by Camilo-Camargo** [iptv-org/community#310] [Show and tell]
I'm developing a graphql api to all iptv-org repositories, here https://github.com/Camilo-Camargo/iptv-api, also I deploy a server at  http://104.208.76.45
I will maintain this api. 
![image](https://user-images.githubusercontent.com/24948289/212350922-1c0e3b62-9c9f-4057-a831-c24744b22552.png)

**Comment by BellezaEmporium** [comment]
Quite interesting project, thank you for proposing it to us.

This may improve certain tasks depending on the end user's needs.
